ape_safe/_cli.py

```diff
@@ -98,22 +98,37 @@ def remove(cli_ctx, alias):
     cls=NetworkBoundCommand, short_help="See pending transactions for a locally-tracked Safe"
 )
 @network_option()
+@click.option("sign_with_local_signers", "--sign", is_flag=True)
+@click.option("--execute", is_flag=False, flag_value=True, default=False)
 @existing_alias_argument(account_type=SafeAccount)
-def pending(network, alias):
+def pending(network, sign_with_local_signers, execute, alias):
     safe = accounts.load(alias)
-    local_signers = set(signer for signer in accounts if signer.address in safe.signers)
-    if local_signers:
-        click.echo("Local Signer(s) detected!")
-        sign_with_local_signers = click.confirm("Do you want to sign unconfirmed transactions")
+    submitter = None
 
-    else:
-        sign_with_local_signers = False
+    if isinstance(execute, str):
+        if execute in accounts.aliases:
+            submitter = accounts.load(execute)
+        elif execute in accounts:
+            submitter = accounts[execute]
+        else:
+            raise AccountsError(f"`--execute` value '{execute}` not found in local accounts.")
+
+    elif execute is True:
+        submitter = safe.local_signers[0]
+
+    for safe_tx, confirmations in safe.pending_transactions():
+        if sign_with_local_signers and len(confirmations) < safe.confirmations_required:
+            pass  # TODO: sign `safe_tx` with local signers not in `confirmations`
```

Comment:
I need this to be implemented to test the part signing the tx you made in the UI I think.
Otherwise, nothing happens and the CLI does not tell me anything, which is also bad.

So we should probably implement this.